index.html

```diff
@@ -4,6 +4,8 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
-
+  <img src = "http://giphy.com/gifs/steph-curry-UMvIlvd2oCisw">
```

Comment:
Ideally don't place spaces around the `=` sign in html. 

However in JavaScript, please do put spaces around the `=` sign.  

So many rules!